src/configuration.rs

```diff
@@ -1,80 +1,77 @@
-use crate::networking::address::*;
+use crate::networking::address::Address;
+use std::net::Ipv4Addr;
 
 pub struct Config {
   /// Default list of host used for peer discovery
-  pub default_hosts: Vec<IPAddress>,
+  pub default_hosts: Vec<Address>,
+  /// from py-ipv8 configuration. UDP socket address.
+  /// There split up in "address" and "port"
+  pub socketaddress: Address,
 }
 
 impl Default for Config {
   fn default() -> Self {
     Config {
+
+      socketaddress:Address{
```

Comment:
```suggestion
      socketaddress: Address{
```

This would make it more readable, now it could be confused with the double colon notation.